Source/JavaScriptCore/wasm/WasmBBQJIT.cpp

```diff
@@ -6290,7 +6645,7 @@ class BBQJIT {
         m_jit.call(m_scratchGPR, OperationPtrTag);
 
         // FIXME: Probably we should make CCall more lower level, and we should bind the result to Value separately.
-        result = Value::fromTemp(returnType, currentControlData().enclosedHeight() + m_parser->expressionStack().size());
+        result = Value::fromTemp(returnType, currentControlData().enclosedHeight() + currentControlData().implicitSlots() + m_parser->expressionStack().size());
```

Comment:
nit: All these modifications just adding `implicitSlots` make me think it'd be nice to have a helper function